codelabs/functions/python_powered/test_main.py

```diff
@@ -0,0 +1,25 @@
+from unittest import TestCase
+from unittest.mock import Mock
+
+from main import hello_name, hello_world
+
+
+class TestHello(TestCase):
```

Comment:
Hmm. I think you're using this in your codelab as example code, yeah?

If so, this is fine, otherwise, we generally use `pytest` style.

Reply:
> Hmm. I think you're using this in your codelab as example code, yeah?

Yeah. I would have used `pytest` style, but the `conftest.py` in this repo seems incompatible with Python 3?

```
$ python -m pytest
ImportError while loading conftest '.../googlecloudplatform/python-docs-samples/conftest.py'.
../../../conftest.py:17: in <module>
    import mock
E   ModuleNotFoundError: No module named 'mock'
```

(It also saves a step for the codelab user here.)